js/build_map.js

```diff
@@ -9,16 +9,21 @@ var projection = albersUsaPr()
     .translate([chart_width / 2, chart_height / 2]);
 var buildPath = d3.geoPath()
     .projection(projection);
+    
+//Create container
+var container = d3.select('body')
+  .append('div')
+  .attr('id', 'container');
 
 // circle scale
 var scaleCircles = d3.scaleSqrt()
   .range([0, 20])
 
 // Create SVG
-var svg = d3.select("body")
+var svg = d3.select("#container")
     .append("svg")
-    .attr("width", chart_width)
-    .attr("height", chart_height);
+    .attr('viewBox', '0 0 ' + chart_width + ' ' + chart_height + '')
+	  .attr('preserveAspectRatio', 'xMidYMid');
```

Comment:
is this indentation intended?

Reply:
Nope

Reply:
But does seem to be a GitHub thing, cause I don't have that indent in my code

Reply:
hmm. tabs versus spaces? i think javascript people might be more tab-centric, but lindsay and i have been tending toward spaces because RStudio will autoconvert tabs to spaces for you and that's the recommended style for R code...